openedx/features/course_experience/static/course_experience/js/currency.js

```diff
@@ -0,0 +1,75 @@
+import whichCountry from 'which-country';
+import 'jquery.cookie';
+import $ from 'jquery'; // eslint-disable-line import/extensions
+import _ from 'underscore'; // eslint-disable-line import/extensions
+
+export class Currency {  // eslint-disable-line import/prefer-default-export
+
+  setCookie(countryCode, l10nData) {
+    const userCountryData = _.pick(l10nData, countryCode);
+    let countryL10nData = userCountryData[countryCode];
+
+    if (countryL10nData) {
+      countryL10nData.countryCode = countryCode;
+    } else {
+      countryL10nData = {
+        countryCode: 'USA',
+        symbol: '$',
+        rate: '1',
+        code: 'USD',
+      };
+    }
+    this.countryL10nData = countryL10nData;
+    $.cookie('edx-price-l10n', JSON.stringify(countryL10nData), {
+      expires: 1,
+    });
+  }
+
+  setPrice() {
+    const l10nCookie = this.countryL10nData;
+    const lmsregex = /(\$)(\d*)( USD)/g;
+    const price = jQuery('input[name="verified_mode"]').filter(':visible')[0];
```

Comment:
Why not use ```$``` here? Isn't that what you define the variable as on line 3?

Reply:
changed